.github/workflows/ci_workable.yml

```diff
@@ -0,0 +1,275 @@
+
+      - name: Cache Go Dependencies
+        uses: actions/cache@v3
+        with:
+          path: ${{ steps.go.outputs.mod-cache }}
+          key: ${{ runner.os }}-mod-${{ github.sha }}-${{ hashFiles('**/go.sum') }}
+
+      # Check DIff also runs Reviewable which needs golangci-lint installed
+      - name: Check Diff
+        run: |
+          wget -O- -nv https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s ${{ env.GOLANGCI_VERSION }}
+          export PATH=$PATH:./bin
+          make check-diff
+
+  unit-tests:
+    runs-on: ubuntu-latest
+    needs: detect-noop
+    if: needs.detect-noop.outputs.noop != 'true' && github.repository == 'Workable/external-secrets'
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Fetch History
+        run: git fetch --prune --unshallow
+
+      - name: Setup Go
+        uses: actions/setup-go@v3
+        with:
+          go-version-file: "go.mod"
+
+      - name: Find the Go Cache
+        id: go
+        run: |
+          echo "::set-output name=build-cache::$(go env GOCACHE)"
+          echo "::set-output name=mod-cache::$(go env GOMODCACHE)"
+
+      - name: Cache the Go Build Cache
+        uses: actions/cache@v3
+        with:
+          path: ${{ steps.go.outputs.build-cache }}
+          key: ${{ runner.os }}-build-${{ github.sha }}-${{ hashFiles('**/go.sum') }}
+
+      - name: Cache Go Dependencies
+        uses: actions/cache@v3
+        with:
+          path: ${{ steps.go.outputs.mod-cache }}
+          key: ${{ runner.os }}-mod-${{ github.sha }}-${{ hashFiles('**/go.sum') }}
+
+      - name: Add setup-envtest
+        run: |
+          go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
+          setup-envtest use ${{env.KUBERNETES_VERSION}} -p env --os $(go env GOOS) --arch $(go env GOARCH)
+
+      - name: Cache envtest binaries
+        uses: actions/cache@v3
+        with:
+          path: /home/runner/.local/share/kubebuilder-envtest/
+          key: ${{ runner.os }}-kubebuilder-${{env.KUBERNETES_VERSION}}
+
+      - name: Run Unit Tests
+        run: |
+          export KUBEBUILDER_ATTACH_CONTROL_PLANE_OUTPUT=true
+          source <(setup-envtest use ${{env.KUBERNETES_VERSION}} -p env --os $(go env GOOS) --arch $(go env GOARCH))
+          make test
+
+  set-ref:
+    if: github.repository == 'Workable/external-secrets'
+    name: retrieve the actual build branch
+    needs:
+      - unit-tests
+    runs-on: ubuntu-latest
+    outputs:
+      ref: ${{ steps.set-branch.outputs.ref }}
+    steps:
+      - name: set the build branch
+        id: set-branch
+        shell: bash
+        run: |
+          if [[ "${{ inputs.manual_build_branch }}" == "" ]]; then
+            ACTUAL_BRANCH=${{ github.ref_name }}
+            echo "ref=${ACTUAL_BRANCH#workable_feature/*}" >> $GITHUB_OUTPUT
+          else
+            echo "ref=${{ inputs.manual_build_branch }}" >> $GITHUB_OUTPUT
+          fi
```

Comment:
Just as a reminder to get the branch and tehn before sending it to $GITHUB_OUTPUT to replace workable_feature/ with ""

Reply:
Aren't I doing this?

```sh
▶ ACTUAL_BRANCH=workable_feature/lalaland

▶ ref=${ACTUAL_BRANCH#workable_feature/*}

▶ echo $ref
lalaland
```